3303Elevator/src/app/FloorSubsystem/FloorSubsystem.java

```diff
@@ -190,13 +230,47 @@ private static String chooseFileWithExplorer(){
         return fileName;
     }
     
-
+    private static void printUIGuidelines() {
+    		 System.err.println("Invalid Input, please use the following guidelines to schedule an elevator request ");
+	 		 System.err.println("1. Enter <Time in milliseconds> or <Timestamp 'hh:mm:ss' eg:22:51:00.00> followed by");
+	 		 System.err.println("2. Enter <CrrentFloor> as a positive non zero number");
+	 		 System.err.println("3. Enter <Direction> as 'Up' or 'Down'");
```

Comment:
Put this all into one string and then print it. Otherwise you might get other prints in between them